spec/bourbon/functions/assign_inputs_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "spec_helper"
+
+describe "assign-inputs" do
+  before(:all) do
+    ParserSupport.parse_file("functions/assign-inputs")
+    @text_inputs_list = [
+      'input[type="password"]',
+      'input[type="text"]',
+      'textarea'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.